app/models/book.rb

```diff
@@ -0,0 +1,3 @@
+class Book < ApplicationRecord
+  validates :gender, :author, :image, :title, :editor, :year, presence: true
```

Comment:
Can you call the validation like: `validates %i[gender author image title editor year], presence: true`

Reply:
Nico, i changed this line but i wonder...what is this? I mean, what means this structure?